test_regress/t/t_vpi_dump_missing_scopes.iv.out

```diff
@@ -0,0 +1,3 @@
+obj_iv/t_vpi_dump_missing_scopes/libvpi.so: Unable to find module file `obj_iv/t_vpi_dump_missing_scopes/libvpi.so' or `obj_iv/t_vpi_dump_missing_scopes/libvpi.so.vpi'.
+*-* All Finished *-*
+t/t_vpi_dump_missing_scopes.v:21: $finish called at 0 (1s)
```

Comment:
I fixed this by allowing the make_pli flag to accept a file name. I think it would also be nice to have TestVpiMain.cpp have a --pli flag that take an .so like other simulators do, and maybe include this as the default main so people could run verilator more similar to how they run other simulators